### ISSUE TITLE
Libspotify no longer available

### ISSUE THREAD

**Issue by j0rdsta** [smashingboxes/maestro#3]
Aw man, Maestro sounds exactly what my workplace needs for a jukebox, but I can't get things installed!

`brew install libspotify` no longer works, because libspotify is ["no longer under active development on any platform and is considered deprecated"](https://developer.spotify.com/technologies/libspotify/). While a replacement has been flagged for '2016', we're in the final weeks of the year and it looks unlikely that Spotify will debut a replacement any time soon.

In the off chance that anyone's reading this:
* Is there any way this project could be rewritten to not use Libspotify?
* Failing that, could I be pointed to a place where I can download a version of Libspotify?

Cheers,
Jordan

**Comment by JohnnyLinnert** [comment]
I second the above -- would love to get Spotify working in my office Slack channel (via Maestro if at all possible!)

**Comment by jgillman** [comment]
@j0rdsta I apparently already had libspotify installed via the [mopidy tap](https://docs.mopidy.com/en/latest/installation/osx/).

You might be able to install like so:

```sh
brew tap mopidy/mopidy
brew install mopidy/mopidy/libspotify
```

**Comment by dkniffin** [comment]
Hey everyone. Sorry we didn't see this sooner. We haven't been doing maintenance on this project for a while now (obviously). We are working on a new version of Maestro, using [shpotify](https://github.com/hnarayanan/shpotify/blob/master/spotify) and Sinatra. We're testing internally first before releasing a Maestro 2.0, but if anyone wants to try it out now, it's available on the shpotify-rewrite branch of this repo.

**Comment by dkniffin** [comment]
Resolved via #6
